app/AppKernel.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Component\Config\Loader\LoaderInterface;
+
+class AppKernel extends Kernel
```

Comment:
__[development][phpmd]__
* The class AppKernel has a coupling between objects value of 17. Consider to reduce the number of dependencies under 13.